spec/rubocop/cop/style/to_json_spec.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Style::ToJson, :config do
+  it 'registers an offense when using JSON.generate' do
+    expect_offense(<<~RUBY)
+      JSON.generate(foo)
+      ^^^^^^^^^^^^^^^^^^ Use `.to_json` instead of `JSON.generate`.
+    RUBY
+
+    expect_correction(<<~RUBY)
+      foo.to_json
+    RUBY
+  end
+
+  it 'recognizes fully qualified reference to JSON' do
+    expect_offense(<<~RUBY)
+      ::JSON.generate(foo)
+      ^^^^^^^^^^^^^^^^^^^^ Use `.to_json` instead of `JSON.generate`.
+    RUBY
```

Comment:
I notice this doesn't have an `expect_correction`. Probably not a huge deal, since the case above basically covers it, but I thought I'd mention it, since I can't recall seeing specs that don't also expect the correction.